modules/oxide/src/less/theme/content/mergetags/mergetag.less

```
@@ -0,0 +1,14 @@
+@mergetag-hover-background-color: rgba(64, 153, 255, .1);
+@mergetag-affix-text-color: #3788EC;
```

It would be better to use a variables for color and background so it would adjust with the user’s skin. Since we don’t seem to have an exact match for the color in the design, Paul suggested using the same color scheme as do tabs in dialog popups.

```suggestion
@mergetag-hover-background-color: fade(@color-tint, 10%);
@mergetag-affix-text-color: @color-tint;
```

---

That makes sense to use existing variables as much as we can here.